templates/app/views/home/_featured_product_banner.html.erb

```diff
@@ -5,9 +5,9 @@
       size: :full,
       class: 'w-full object-cover aspect-square md:aspect-auto'
     )) %>
-    <div class="absolute bottom-0 left-0 right-0 text-center pb-14">
-      <span class="rounded-full px-3 py-1 bg-white text-sm text-black mb-2">New</span>
-      <h4 class="text-h3 font-serif-md mb-6 text-white md:mb-11 md:text-h2.5"><%= product.name %></h4>
+    <div class="grid-rows-3	absolute bottom-0 left-0 right-0 text-center pb-14">
```

Comment:
These changes seem very opinionated and maybe belong into your own application instead of this template. Can you explain why you put them into this PR. I how are they related to the overall goal of removing heading tags?

Reply:
Hey, @tvdeyen the sentiment on my side was preserving the entire style and only change http tags.
On a few edge cases a simple transistion to stack would have changed the layout. I chose to go with a vertical grid as it allowed an identical display of the information as before. 
<img width="1019" alt="Screenshot 2024-11-14 at 19 53 55" src="https://github.com/user-attachments/assets/3ae9751b-2e17-4493-851b-90544137fa67">

Reply:
@tvdeyen Do you want to refactor the div or do you want to keep it like this?